index.html

```diff
@@ -32,40 +32,295 @@
+            <li>Canopy</li>
+            <li>&#x2022;</li>
+            <li>Back End Dev</li>
+            <li>&#x2022;</li>
+            <li>2015</li>
+          </ul>
+          <span
+            >A daily selection of privately personalized reads; no accounts or
+            sign-ups required.</span
+          >
+          <ul class="technologies">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <div id="button-container">
+            <button type="button" class="btn">
+              <a href="#">See Project</a>
+            </button>
+          </div>
+        </li>
+        <li class="work-card">
+          <img
+            src="images/work_3.jpg"
+            alt="work_tonic_daily_personalized_reads"
+          />
+          <h2>Tonic</h2>
+          <ul class="experience">
+            <li>Canopy</li>
+            <li>&#x2022;</li>
+            <li>Back End Dev</li>
+            <li>&#x2022;</li>
+            <li>2015</li>
+          </ul>
+          <span
+            >A daily selection of privately personalized reads; no accounts or
+            sign-ups required.</span
+          >
+          <ul class="technologies">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <div id="button-container">
+            <button type="button" class="btn">
+              <a href="#">See Project</a>
+            </button>
+          </div>
+        </li>
+        <li class="work-card">
+          <img
+            src="images/work_4.jpg"
+            alt="work_multi-post_stories_daily_personalized_reads"
+          />
+          <h2>Multi-Post Stories</h2>
+          <ul class="experience">
+            <li>Canopy</li>
+            <li>&#x2022;</li>
+            <li>Back End Dev</li>
+            <li>&#x2022;</li>
+            <li>2015</li>
+          </ul>
+          <span
+            >A daily selection of privately personalized reads; no accounts or
+            sign-ups required.</span
+          >
+          <ul class="technologies">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <div id="button-container">
+            <button type="button" class="btn">
+              <a href="#">See Project</a>
+            </button>
+          </div>
+        </li>
+      </ul>
+    </section>
+    <section id="about-container">
+      <ul id="about">
+        <li>
+          <h2 class="header">
+            <span>About<br />Myself</span>
+          </h2>
+        </li>
+        <li>
+          <p class="text">
+            I’m a software developer! I can help you build a product, feature or
+            website. Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+        </li>
+        <li>
+          <span id="connect">Let's Connect</span>
```

Comment:
- [ ] Your `Let's Connect` title should be a header `hx` instead of a `span`. Please fix that by using a `h2` tag. 👌